### ISSUE TITLE
Seg3D fails to start under Windows 10

### ISSUE THREAD

**Issue by robertf** [SCIInstitute/Seg3D#397]
### Description

Seg3D 2.5.0 fails to start under Windows 10

### Steps to Reproduce

1. Download Seg3D2-2.5.0-win64.exe to Windows 10
2. Run installer, apparently successfully
3. Run Seg3D2  using its icon in the Start Menu (or from the command line). I also tried running it as Administrator.

**Expected behavior:** It should start

**Actual behavior:** An Application Error window appeared saying that the application failed to start (0xc000007b)

### Versions

Windows 10 Home 20H2 19042.804 120.2212.551.0

**Comment by dcwhite** [comment]
I'll try to replicate this one. It might be the out-of-date Qt libraries used.

**Comment by allywarner** [comment]
I can't replicate this bug even after updating my Windows OS version. Can you replicate @dcwhite?

**Comment by allywarner** [comment]
Try with new release 2.5.1

**Comment by robertf** [comment]
Is 2.5.1 available yet? I couldn't find it.

**Comment by allywarner** [comment]
@robertf yes! It is now posted! Thank you for your patience.

**Comment by robertf** [comment]
It still fails to start, the same as before :-(
C:\Program Files\Seg3D2_2.5\bin\Seg3D2.exe with timestamp 2021-05-21 16:37

**Comment by allywarner** [comment]
What kind of machine are you running?

**Comment by robertf** [comment]
ASUS VivoBook Pro 15 N580GD-DB74, 64-bit, 16 GB

**Comment by allywarner** [comment]
Do you have another machine to try on?

**Comment by robertf** [comment]
I have an old Windows 7 laptop that might wake up long enough to try it.

**Comment by allywarner** [comment]
Let me know what happens!

**Comment by robertf** [comment]
It can't start because MSVCP140_1.DLL is missing.

**Comment by allywarner** [comment]
Yes, that's probably too old. We can't reproduce this with our machines. Is there any information when you try to open Seg3D on Windows 10? Possibly on the command line?

**Comment by robertf** [comment]
When I give this command in the PowerShell:
& "C:\Program Files\Seg3D2_2.5\bin\Seg3D2.exe"
nothing happens, it just returns to the command prompt.

**Comment by robertf** [comment]
I also tried running it from the Start menu as administrator, with the same result. I've attached a screenshot of the error-message window.
![Seg3D2_error](https://user-images.githubusercontent.com/1523712/119208334-c0941b00-ba6f-11eb-9427-e953a576735c.png)

**Comment by allywarner** [comment]
Is your machine in French?

**Comment by robertf** [comment]
Partly. When the shop installed Windows 10, they did it in French. When I added my account I did it in English, and the interface is in English, but occasionally bits of French still pop up for reasons I don't understand. It's never caused a problem.

**Comment by dcwhite** [comment]
https://www.diskgenius.com/how-to/fix-error-code-0xc000007b.php

Is your machine running 32-bit Windows perhaps? I suppose it's possible with the complicated setup you described.

**Comment by dcwhite** [comment]
That web page also mentions updating DirectX, .NET Framework, C++ runtime packages, and various other things. Essentially, this is most likely a machine problem rather than a Seg3D problem.

**Comment by robertf** [comment]
It's definitely 64-bit and Windows is fully up to date.
.NET Framework 4.8 was installed but not .NET Framework 3.5 so I installed it and then rebooted.
Still no luck :-(
I'll let you know if I find anything that helps.
